yarn-project/sequencer-client/src/block_builder/solo_block_builder.test.ts

```diff
@@ -157,11 +159,11 @@ describe('sequencer/solo_block_builder', () => {
   const updateArchive = async () => {
     const blockHash = computeBlockHashWithGlobals(
```

Comment:
We should probably come back to what the block hash here actually is.

Reply:
Agree, created an issue for it and linked it in the code [here](https://github.com/AztecProtocol/aztec-packages/pull/3827/commits/73b4b69941f77920bc7b9ee4f36d71c2d0bf6372).